rotkehlchen/api/v1/schemas.py

```diff
@@ -1008,7 +1010,7 @@ def make_history_base_entry(
                 identifier=data.get('identifier'),
                 amount=data['balance'].amount,
                 extra_data=extra_data,
-                fee_identifier=self.context['schema'].get_fee_event_identifier(data),
+                fee_identifier=CreateHistoryEventSchema.history_event_context.get()['schema'].get_fee_event_identifier(data),
```

Comment:
cant't you use self here? We use self for all other class variables such as `include_identifier`

```suggestion
                fee_identifier=self.history_event_context.get()['schema'].get_fee_event_identifier(data),
```